doc-Monitoring_Alerts_and_Reporting/topics/To_manage_report_menu_folders.adoc

```diff
@@ -7,12 +7,12 @@
 . Click the role whose menus you want to customize.
 . Click on the accordion name you want to organize or add folders to.
 +
-* Click  image:2305.png[] *(Move selected folder to top)* to move the folder to the top of the list.
-* Click  image:2290.png[] *(Move selected folder up)* to move the folder up.
-* Click  image:2289.png[] *(Move selected folder down)* to move the folder down.
-* Click  image:2306.png[] *(Move selected folder to bottom)* to move the folder to the bottom of the list.
-* Click  image:1888-.png[] *(Delete selected folder and its contents)* to delete an folder.
-* Click  image:1848.png[] *(Add subfolder to selected folder)* to add a folder.
+* Click  image:moveToTop.png[] *(Move selected folder to top)* to move the folder to the top of the list.
+* Click  image:moveUP.png[] *(Move selected folder up)* to move the folder up.
```

Comment:
I'm not sure why, but this icon doesn't show for me in the preview. Does it build OK locally and is the icon checked in?  "Click moveUP (Move selected folder up) to move the folder up. " I wonder if it is case sensitive?